src/views/virtualmachines/actions/components/CloneVMModal/utils/helpers.tsx

```diff
@@ -155,21 +155,20 @@ export const updateClonedDataVolumes = (
   vm: V1VirtualMachine,
   pvcs: IoK8sApiCoreV1PersistentVolumeClaim[],
 ): V1VirtualMachine => {
+  const randomChars = getRandomChars(5);
```

Comment:
can you please add a `-` to the beginning of these random chars?
the name comes ...clonefsdkj which would look more readable if it was ...clone-fsdkj